library/src/main/java/com/ricohapi/mstorage/RicohAPIRequest.java

```diff
@@ -36,13 +36,15 @@ public RicohAPIRequest(String urlString, Map<String, String> queryParams) throws
 
     public void get(Map<String, String> header) throws IOException {
         connection.setRequestMethod("GET");
+        connection.setDoOutput(false);
```

Comment:
It's not necessary, but I wanted to make sure that the request won't change into "POST" method.